recipes/librdkafka/all/patches/0002-Change-library-targets-and-result-variables-1-5-0.patch

```diff
@@ -0,0 +1,48 @@
+--- src/CMakeLists.txt	(revision 39796d359898c07ea422849e6d7cd34cd13ec466)
++++ src/CMakeLists.txt	(date 1596441634092)
+@@ -209,9 +209,9 @@
+ endif()
+ 
+ if(WITH_ZSTD)
+-  target_link_libraries(rdkafka PUBLIC ${ZSTD_LIBRARY})
+-  target_include_directories(rdkafka PUBLIC ${ZSTD_INCLUDE_DIR})
+-  message(STATUS "Found ZSTD: ${ZSTD_LIBRARY}")
++  target_link_libraries(rdkafka PUBLIC ${zstd_LIBRARY})
++  target_include_directories(rdkafka PUBLIC ${zstd_INCLUDE_DIR})
++  message(STATUS "Found ZSTD: ${zstd_LIBRARY}")
+ endif()
+ 
+ if(WITH_SSL)
+@@ -224,8 +224,8 @@
+     add_dependencies(rdkafka bundled-ssl)
+   else()
+     find_package(OpenSSL REQUIRED)
+-    target_include_directories(rdkafka PUBLIC ${OPENSSL_INCLUDE_DIR})
+-    target_link_libraries(rdkafka PUBLIC OpenSSL::SSL OpenSSL::Crypto)
++    target_include_directories(rdkafka PUBLIC ${OpenSSL_INCLUDE_DIR})
++    target_link_libraries(rdkafka PUBLIC OpenSSL::OpenSSL)
```

Comment:
Just trying to understand if there is a reason, why moving from components to the global target?

Reply:
Not sure why - this patch file was carried over from 1.4.0.  I'll try applying this and the other comments this weekend.